tasks/configure-OpenBSD.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+# XXX rpc.lockd(8) of OpenBSD does not support client-side of the protocol
```

Comment:
these days, most, if not all, of distributions can `mount -t nfs` by default. what you would expect an NFSv3 role to do is to configure locking. you do not need an `ansible` role ro mount NFS file system. if you do not need locking, there is no reason to use the role. what else should be commented?